### PR TITLE
Makes a really shitty lord of the rings reference

### PR DESCRIPTION
**Pull request by KazeEspada**
THROW THE GREENTEXT INTO THE FIRE!
https://www.youtube.com/watch?v=ajUlhaX9hQI

### CODE REVIEW

**Comment by Shadowlight213** [comment]
Reference policy!!

**Comment by KazeEspada** [comment]
Reference policy prevents direct thievery and or references to players or people.
This is subtle enough to be a hint towards the one ring without blatantly saying it.

File: code/modules/events/wizard/greentext.dm [hunk 1]
```diff
@@ -75,6 +76,8 @@
 		last_holder = new_holder //long live the king
 
 /obj/item/weapon/greentext/Destroy()
+	if(burn_state != ON_FIRE)
```
Comment (optimumtact) [inline code/modules/events/wizard/greentext.dm]: What is this doing?

Comment (KazeEspada) [inline code/modules/events/wizard/greentext.dm]: If its not on fire, than do not destroy(), otherwise destroy it!

Comment (optimumtact) [inline code/modules/events/wizard/greentext.dm]: I see

Comment (KazeEspada) [inline code/modules/events/wizard/greentext.dm]: Only by lava.

**Comment by Core0verload** [comment]
Singulo > lava.

File: code/modules/events/wizard/greentext.dm (hunk 1 above)
Comment (optimumtact) [inline code/modules/events/wizard/greentext.dm]: cool :+1:

**Comment by coiax** [comment]
So what happens if it falls into a chasm?

**Comment by octareenroon91** [comment]
Chasm dgaf. Maybe add awareness of greentext entering nullspace.

**Pull request merged**